src/utils/cacheManagement.ts

```diff
@@ -0,0 +1,32 @@
+import { Client, ForumChannel, TextChannel, ThreadChannel } from "discord.js";
+
+export async function updateCache(client: Client) {
+    // Iterate through channels in the cache
+    client.channels.cache.forEach(async (channel) => {
+        try {
+            // If it's a ForumChannel
+            if (channel instanceof ForumChannel) {
+                // Fetch all its threads
+                const threads = await channel.threads.fetch()
+                // Iterate the threads
+                threads.threads.forEach(async (thread) => {
+                    // Ensure it's really a ThreadChannel, partially to satisfy TypeScript
```

Comment:
😂

Reply:
Sometimes I love Typescript, sometimes I'm just annoyed by it :sweat_smile: